Update Windows Installation Prerequisite

## Environment details

- OS: Windows 11 x64
- OS version: 23H2 build 22631.2715
- node-pty version: `1.0.0`

## Issue description

While installing with npm under node v21.4.0
```PowerShell
npm install -g node-pty
```
It failed and produced the following messages for 3 `.vcxproj` files, namely: `conpty`, `conpty_console_list`, and `pty`
```log
npm ERR! C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Microsoft\VC\v170\Microsoft.CppBuild.targets(504,5): error MSB8040: Spectre-mitigated libraries are required for this project. Install them from the Visual Studio installer (Individual components tab) for any toolsets and architectures being used. Learn more: https://aka.ms/Ofhn4c [<my nodejs path>\bin\node_modules\node-pty\build\conpty.vcxproj]
```
Luckily, I was able to resolve this with installing `MSVC v143 - VS 2022 C++ x64/x86 Spectre-mitigated libs (v14.38-17.8)`
![image](https://github.com/microsoft/node-pty/assets/11015675/2aeae2aa-a123-484b-8dbf-ae498fb5e5ac)

Thus, the [Windows Installation Prerequisite](https://github.com/microsoft/node-pty#windows) description is quite outdated.
>  [Windows SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk) - only the "Desktop C++ Apps" components are needed to be installed
1. Not only `Desktop Development with C++` under "Workloads" tab, but also the corresponding Spectre-mitigated libs under "Individual components" tab are needed
2. (Optional) https://www.npmjs.com/package/windows-build-tools is not needed for newer Node.js env 
```log
npm WARN deprecated windows-build-tools@5.2.2: Node.js now includes build tools for Windows. You probably no longer need this tool. See https://github.com/felixrieseberg/windows-build-tools for details.`
```

--- a/README.md
+++ b/README.md
@@ -104,6 +104,7 @@ npm install --global --production windows-build-tools
 The following are also needed:
 
 - [Windows SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk) - only the "Desktop C++ Apps" components are needed to be installed
+- Spectre-mitigated libraries - In order to avoid the build error "MSB8040: Spectre-mitigated libraries are required for this project", open the Visual Studio Installer, press the Modify button, navigate to the "Individual components" tab, search "Spectre", and install an option like "MSVC v143 - VS 2022 C++ x64/x86 Spectre-mitigated libs (Latest)" (the exact option to install will depend on your version of Visual Studio as well as your operating system architecture)
 
 ## Debugging
 

